Topology/README.md

```diff
@@ -28,10 +29,56 @@ The following are optional fields and default values (option, more a list of pos
     - `doi` - A doi reference for the molecule.
 
 Other possible quantities:
-  - Bonds - Holding data for MM computations
   - Basis Sets per atom 
   - `fix_com` (bool) - whether to adjust to the molecule to the COM or not
   - `fix_orientation` (bool) - whether to rotate the molecule to a standard orientation or not
   - label (list of str) - Per-atom labels which may be seperate from fragments
   - Extend the `real` quantitity to cover real, ghost, absent, qm/mm region, etc.
   - EFP quantities `fragment_types`, `coordinate_hints`. This is an example and likely not part of the spec. How would we handle this? 
+
+# Connectivity
+
+Connectivity describes relationships between lists of atoms and associated metadata: typically, this is between a pair of atoms (a bond). Connectivity is supported across periodic boundaries.
+
+Each connection is defined as:
+
+* `indices`, an ordered list of length `N` of atomic indexes defining the connection
+* `images`, (optional, only relevant if a unit cell lattice is defined) an ordered list of length `3N` describing the periodic image of the connected atom -- by default, this is assumed to be (0,0,0), meaning that the connected atom is in the origin image, and first index is set to (0,0,0) by convention
```

Comment:
Why not a list of lists like for `gemoetry` and `unit_cell`?

Reply:
I'd like that -- it'd make it a lot more human readable. I'm not sure if it'd have consequences for file size/loading time for large systems though.

Reply:
Right, but `geometry` is currently specified as a list of lists, and the same issue applies there...

Reply:
I agree it should be consistent with `geometry`, I didn't realize this was list of lists.